javagen/src/main/java/com/azure/autorest/mapper/ClientParameterMapper.java

```diff
@@ -52,15 +52,22 @@ public ClientMethodParameter map(Parameter parameter) {
         }
         builder.isConstant(isConstant).defaultValue(defaultValue);
 
-        if (parameter.getSchema() != null && parameter.getSchema().getLanguage() != null) {
-            String description = parameter.getSchema().getLanguage().getDefault().getDescription();
-            if (description == null || description.isEmpty()) {
-                if (parameter.getLanguage() != null) {
-                    description = parameter.getLanguage().getDefault().getDescription();
-                }
+        String description = null;
+        // parameter description
```

Comment:
The original logic prioritizes the parameter schema's description over the parameter's own description. I feel like it should be the other way around

Reply:
Agree that the parameter's description should take precedence.